packages/amplication-server/src/core/git/dto/args/BaseGitArgs.ts

```diff
@@ -4,7 +4,9 @@ import { EnumSourceControlService } from '../enums/EnumSourceControlService';
 @ArgsType()
 export class BaseGitArgs {
```

Comment:
I don't think this class is helpful, and it is probably better to use the fields when needed directly in the class